.github/workflows/tf-deploy-basic.yaml

```
@@ -1,42 +1,64 @@
 on:
   workflow_call:
     inputs:
-      account_id:
+      artifact_path:
+        description: "The path of the artifacts to download"
         type: string
-        required: true
-      environment:
+        required: false
+        default: ""
+      aws_account_id:
+        description: "The AWS account ID"
         type: string
         required: true
-      feedback:
-        required: false
-        type: boolean
-        default: false
-      region:
+      aws_region:
+        description: "The AWS region"
         required: true
         type: string
-      role:
+      aws_role_name:
+        description: "The name of the role to assume with OIDC"
         type: string
         required: true
         default: cicd-iac
-      state_file:
-        required: false
+      environment:
+        description: "The environment name"
         type: string
-      state_bucket:
+        required: true
+      tf_deploy_override:
+        description: "If true, terraform apply will run regardless of branch"
+        type: boolean
         required: false
-        type: string
-      terraform_dir:
+        default: false
+      tf_dir:
         description: "The directory where the Terraform files are located"
         type: string
         required: false
         default: .
-      terraform_version:
-        description: "The Terraform version to use"
+      tf_feedback:
```

```suggestion
      github_feedback:
```

so it could be more generic than only for terraform plan